portal/src/main/webapp/js/src/civic/civicservice.js

```diff
@@ -0,0 +1,166 @@
+
+function CivicService() {
+    
+    // All results are stored here
+    var civicGenes = {};
+    
+    function getCivicGenesBatch(ids) {
+        return $.ajax({
+            type: 'GET',
+            url: 'api-legacy/proxy/civic/genes/' + ids,
```

Comment:
We tried, but the civic api doesn't support cross origin resource sharing, which forces us to use the proxy.